contracts/multisig_wallet.scilla

```diff
@@ -77,7 +77,7 @@ type SsnRewardShare =
 
 (* Type of Proxy transactions. *)
 (* All calls are made to the proxy contract *)
-type ProxyTransaction =
+type CalleeTransaction =
 (***************************************************)
 (*               Proxy Transition                  *)
```

Comment:
Rename this?

Reply:
```suggestion
(*               Transitions for the callee contract                  *)
```

Reply:
I think this is ok because this part list the transitions that are in the proxy contract.

Reply:
I think this is ok. This part lists transition that are only for proxy contract, i.e., transitions that do not lead to a call to the SSNList contract or gZIL contract.

Reply:
oh, i changed this because we have:
```
(***************************************************)
(*              gZIL Transitions                   *)
(***************************************************)
(* ChangeMinter(new_minter: ByStr20) *)
| ChangeMinter of ByStr20
```

So the comment might need to be changed too.